src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const day = 40;
```

Comment:
Let's create more meaningful name for this:
```suggestion
  const oneDayCost = 40;
```